packages/cli/src/utils/schema.ts

```diff
@@ -0,0 +1,47 @@
+import { getStaticByteLength, SchemaType } from "@latticexyz/schema-type";
+import { MUDError } from "./errors.js";
+
+/**
+ * Encode a table schema into a bytes32 hex string
+ * Port of `Schema.sol` from `@latticexyz/store`
+ * @param schema The schema to encode SchemaType[]
+ * @returns The encoded schema as a 32 byte hex string
+ */
+export function encodeSchema(schema: SchemaType[]): string {
```

Comment:
solidity one uses Bytes extensively, and I wouldn't wanna duplicate that in schema-type.
I think it could be useful to make a `solidity-utils` first, put `Bytes`, `Memory` and maybe some others in there, and then use that in `schema-type` to do schema encoding